TodoListClient/MainWindow.xaml.cs

```diff
@@ -141,7 +141,7 @@ public MainWindow()
             AuthenticationResult result = null;
             try
             {
-                result = await app.AcquireTokenAsync(new string[] { clientId });
+                result = await app.AcquireTokenSilentAsync(new string[] { clientId });
```

Comment:
I'm guessing the app should be attempting to acquire the token silently at this point as the error handling code in line 148 below is testing for _failed_to_acquire_token_silently_, and the current app behaviour is odd: the login dialogue appears every time the user clicks on _Add item_.
I'll update the _complete_ branch if this change is correct.